scripts/us_census/acs5yr/subject_tables/common/testdata/spec.json

```diff
@@ -0,0 +1,372 @@
+		"Margin of Error!!Total Uninsured!!Total civilian noninstitutionalized population",
+		"Estimate!!Total Uninsured!!RESIDENCE 1 YEAR AGO!!Civilian noninstitutionalized population 1 year and over",
+		"Margin of Error!!Total Uninsured!!RESIDENCE 1 YEAR AGO!!Civilian noninstitutionalized population 1 year and over",
+		"Estimate!!Total Uninsured!!EDUCATIONAL ATTAINMENT!!Civilian noninstitutionalized population 25 years and over",
+		"Margin of Error!!Total Uninsured!!EDUCATIONAL ATTAINMENT!!Civilian noninstitutionalized population 25 years and over",
+		"Estimate!!Total Uninsured!!EMPLOYMENT STATUS!!Civilian noninstitutionalized population 16 years and over",
+		"Margin of Error!!Total Uninsured!!EMPLOYMENT STATUS!!Civilian noninstitutionalized population 16 years and over",
+		"Estimate!!Total Uninsured!!WORK EXPERIENCE!!Civilian noninstitutionalized population 16 to 64 years",
+		"Margin of Error!!Total Uninsured!!WORK EXPERIENCE!!Civilian noninstitutionalized population 16 to 64 years",
+		"Estimate!!Total Uninsured!!Civilian noninstitutionalized workers 16 years and over",
+		"Margin of Error!!Total Uninsured!!Civilian noninstitutionalized workers 16 years and over",
+		"Estimate!!Total Uninsured!!EARNINGS IN THE PAST 12 MONTHS (IN 2019 INFLATION ADJUSTED DOLLARS)!!Civilian noninstitutionalized population 16 years and over with earnings",
+		"Margin of Error!!Total Uninsured!!EARNINGS IN THE PAST 12 MONTHS (IN 2019 INFLATION ADJUSTED DOLLARS)!!Civilian noninstitutionalized population 16 years and over with earnings",
+		"Estimate!!Total Uninsured!!HOUSEHOLD INCOME (IN 2019 INFLATION ADJUSTED DOLLARS)!!Total household population", "Margin of Error!!Total Uninsured!!HOUSEHOLD INCOME (IN 2019 INFLATION ADJUSTED DOLLARS)!!Total household population",
+		"Estimate!!Total Uninsured!!RATIO OF INCOME TO POVERTY LEVEL IN THE PAST 12 MONTHS!!Civilian noninstitutionalized population for whom poverty status is determined",
+		"Margin of Error!!Total Uninsured!!RATIO OF INCOME TO POVERTY LEVEL IN THE PAST 12 MONTHS!!Civilian noninstitutionalized population for whom poverty status is determined"
+	],
+	"denominators": {
+		"Estimate!!Total Uninsured!!Total civilian noninstitutionalized population": [
+			"Under 19 years",
+			"Under 6 years",
+			"6 to 18 years",
+			"19 to 64 years",
+			"19 to 25 years",
+			"26 to 34 years",
+			"35 to 44 years",
+			"45 to 54 years",
+			"55 to 64 years",
+			"65 years and older",
+			"65 to 74 years",
+			"75 years and older",
+			"Male",
+			"Female",
+			"One race",
+			"White alone",
+			"Black or African American alone",
+			"American Indian and Alaska Native alone",
+			"Asian alone",
+			"Native Hawaiian and Other Pacific Islander alone",
+			"Some other race alone",
+			"Two or more races",
+			"Hispanic or Latino (of any race)",
+			"White alone, not Hispanic or Latino",
+			"Native born",
+			"Foreign born",
+			"Naturalized",
+			"Not a citizen",
+			"With a disability",
+			"No disability"
```

Comment:
I'm not sure if there are edge-cases with values not being full columns.

(I don't want to belabor the PR though, we can add a TODO and revisit.)